test/WebSites/ActionConstraintsWebSite/Startup.cs

```diff
@@ -21,7 +21,7 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
-            var configuration = app.GetTestConfiguration();
+            app.GetTestConfiguration();
```

Comment:
So what does this do now? presumably magic? Can we name it accordingly `app.AddTestServices()` or something reasonable?